ontology/vario.md

```diff
@@ -7,15 +7,48 @@ build:
   source_url: http://variationontology.org/vario_download/vario.obo
 contact:
   email: mauno.vihinen@med.lu.se
+  github: maunov
   label: Mauno Vihinen
+  orcid: 0000-0002-9614-7976
 description: Variation Ontology, VariO, is an ontology for standardized, systematic description of effects, consequences and mechanisms of variations.
 domain: biological systems
 homepage: http://variationontology.org
+license:
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
 preferredPrefix: VariO
 products:
 - id: vario.owl
+  title: VariO in OWL Functional notation
+- id: vario.owx
+  title: VariO in OWL RDF/XML
 - id: vario.obo
-activity_status: orphaned
+  title: VariO in OBO
+publications:
+- id: https://www.ncbi.nlm.nih.gov/pubmed/24162187
+  title: Variation Ontology for annotation of variation effects and mechanisms
+- id: https://www.ncbi.nlm.nih.gov/pubmed/24533660
+  title: 'Variation ontology: annotator guide'
+- id: https://www.ncbi.nlm.nih.gov/pubmed/25616435
+  title: Types and effects of protein variations
+activity_status: inactive
 ---
 
-Variation Ontology, VariO, is an ontology for standardized, systematic description of effects, consequences and mechanisms of variations.
+## Requirements to become "active"
```

Comment:
@matentzn should we keep this "todo" list in here? or should I remove this? Either way I think we should merge this PR without waiting for these updates, since they might take some time